web/webkit/src/main/scala/net/liftweb/http/provider/servlet/HTTPRequestServlet.scala

```diff
@@ -234,8 +236,8 @@ private class OfflineRequestSnapshot(req: HTTPRequest, val provider: HTTPProvide
 
   val scheme: String = req.scheme
 
-  lazy val serverPort: Int = req.serverPort match {
-    case 80 => headers("X-SSL").flatMap(Helpers.asBoolean _).filter(a => a).map(a => 443).headOption getOrElse 80
+  lazy val serverPort: Int = _serverPort match {
+    case 80 => headers("X-SSL").flatMap(Helpers.asBoolean).filter(identity).map(a => 443).headOption getOrElse 80
```

Comment:
While we're here, can we change this to be a bit more readable?

```scala
case 80 =>
  headers("X-SSL")
    .flatMap(Helpers.asBoolean)
    .filter(identity)
    .map(a => 443)
    .headOption
    .getOrElse(80)
```

Also, what's the point of the `filter`?